### PR TITLE
Runtime upgrade simulation

### LINKED ISSUE #378: GA to integrate the runtime upgrade simulation scripts
see parent epic

### PR DESCRIPTION
**Pull request by Kailai-Wang**
resolves #378 

This PR:
- adds script to fork off parachain and launch a local network
- adds GHA to test runtime upgrade whenever a release is published
- prettier

### CODE REVIEW

**Review by Marky-Shi — APPROVED** [review]
Thanks!  looks very good

File: ts-tests/tests/runtime-upgrade.test.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    await api.tx.parachainSystem.enactAuthorizedUpgrade(wasm).signAndSend(alice, { nonce: -1 });
+    console.log('Submitted enactAuthorizedUpgrade');
+
+    // wait for 10 blocks
+    console.log('Wait for new runtime ...');
+    currentBlock = (await api.rpc.chain.getHeader()).number.toNumber();
+    let timeoutBlock = currentBlock + 10;
+    let runtimeUpgraded = false;
+
+    return new Promise(async (resolve, reject) => {
```
Comment (jingleizhang) [inline ts-tests/tests/runtime-upgrade.test.ts:37]: Although the whole return value here is a little bit hard to understand, it works fine.

**Review by jingleizhang — APPROVED** [review]
LGTM

File: ts-tests/tests/runtime-upgrade.test.ts (hunk 1 above)
Comment (jingleizhang) [reply ts-tests/tests/runtime-upgrade.test.ts:37]: https://github.com/litentry/litentry-parachain/blob/d714d251661e8bb3c534ca6827c90c44fef44790/ts-tests/tests/runtime-upgrade.test.ts#L37-L61

**Pull request merged**